### PR TITLE
Add help to two new POSM pages

### PR DESCRIPTION
**Pull request by scottcwilson**

### CODE REVIEW

File: zc_plugins/POSM/v6.0.0/admin/includes/classes/observers/auto.PosmHelp.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+class zcObserverPosmHelp extends base {
+  function __construct() {
+    $this->attach($this, array('NOTIFIER_PLUGIN_HELP_PAGE_URL_LOOKUP'));
```
Comment (lat9) [inline zc_plugins/POSM/v6.0.0/admin/includes/classes/observers/auto.PosmHelp.php:4]: Use short-array syntax.

File: zc_plugins/POSM/v6.0.0/admin/includes/classes/observers/auto.PosmHelp.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  }
+
+  function update(&$class, $eventID, $page, &$help_page) {
+     if ($eventID == 'NOTIFIER_PLUGIN_HELP_PAGE_URL_LOOKUP') {
```
Comment (lat9) [inline zc_plugins/POSM/v6.0.0/admin/includes/classes/observers/auto.PosmHelp.php:8]: There's only one event being monitored, no need to check the `$eventID`.

File: zc_plugins/POSM/v6.0.0/admin/includes/classes/observers/auto.PosmHelp.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        if ($page == FILENAME_PRODUCTS_OPTIONS_STOCK) {
+           $help_page = 'https://docs.zen-cart.com/user/running/posm/'; 
+        } elseif ($page == FILENAME_PRODUCTS_OPTIONS_STOCK_VIEW_ALL) {
```
Comment (lat9) [inline zc_plugins/POSM/v6.0.0/admin/includes/classes/observers/auto.PosmHelp.php:11]: It's the same help-page for both; use in_array to see if `$page` is either and set the help-page if true.

File: zc_plugins/POSM/v6.0.0/admin/includes/classes/observers/auto.PosmHelp.php [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<?php
+class zcObserverPosmHelp extends base {
```
Comment (lat9) [inline zc_plugins/POSM/v6.0.0/admin/includes/classes/observers/auto.PosmHelp.php:2]: Curly-brace on next line.

File: zc_plugins/POSM/v6.0.0/admin/includes/classes/observers/auto.PosmHelp.php [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  function __construct() {
```
Comment (lat9) [inline zc_plugins/POSM/v6.0.0/admin/includes/classes/observers/auto.PosmHelp.php:3]: Curly-brace on next line.  Add `public` to function declaration.

File: zc_plugins/POSM/v6.0.0/admin/includes/classes/observers/auto.PosmHelp.php [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    $this->attach($this, array('NOTIFIER_PLUGIN_HELP_PAGE_URL_LOOKUP'));
+  }
+
+  function update(&$class, $eventID, $page, &$help_page) {
```
Comment (lat9) [inline zc_plugins/POSM/v6.0.0/admin/includes/classes/observers/auto.PosmHelp.php:7]: Curly-brace on next line; add `protected` to function declaration.

**Review by lat9 — CHANGES_REQUESTED** [review]
... and shouldn't the file be named `auto.posm_help.php`?

**Comment by lat9** [comment]
Also, use 4-space indentation for class' methods.

**Comment by scottcwilson** [comment]
Updated.

File: zc_plugins/POSM/v6.0.0/admin/includes/classes/observers/auto.PosmHelp.php [hunk 7]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+class zcObserverPosmHelp extends base
+{
+    function __construct()
```
Comment (lat9) [inline zc_plugins/POSM/v6.0.0/admin/includes/classes/observers/auto.PosmHelp.php:5]: Add `public` visibility to method

File: zc_plugins/POSM/v6.0.0/admin/includes/classes/observers/auto.PosmHelp.php [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+    {
+        $this->attach($this, ['NOTIFIER_PLUGIN_HELP_PAGE_URL_LOOKUP']);
+    }
+
+    function update(&$class, $eventID, $page, &$help_page)
```
Comment (lat9) [inline zc_plugins/POSM/v6.0.0/admin/includes/classes/observers/auto.PosmHelp.php:10]: Add `protected` visibility to method.

**Review by lat9 — COMMENTED** [review]
Again, please change the name of the auto-loaded observer to match the remainder of the core offerings: `auto.posm_help.php`.

**Review by lat9 — APPROVED** [review]

**Pull request merged**